Makefile

```diff
@@ -1,7 +1,7 @@
 include .bingo/Variables.mk
 
-OS                    ?= $(shell uname -s | tr '[A-Z]' '[a-z]')
-ARCH                  ?= $(shell uname -m)
+OS                    ?= linux
```

Comment:
Why are we hard-coding these? Without these parameters `go` tool itself pretty clever to determine the os and architecture by given these default you introduce a regression for user. They need to specify those to build. This is not what we want.

Reply:
I, for example, use macbook for development and if I build container without specifying OS, I get `darwin`. This leads to runtime error as my drone uses linux to execute pipelines. I really doubt that anyone would do it on macos, but for those who would, goreleaser will make correct images. 
As for ARCH, `uname` returns `x86_64` instead of `amd64`. It leads to error when building container.
As far as I understand, the build command is used only in drone pipeline to ensure that build is not broken and for local build and development. For these purposes these values can and should be hardcoded.